src/actions/make/make_makefile.lua

```diff
@@ -0,0 +1,93 @@
+--
+-- make_makefile.lua
+-- Generate a C/C++ project makefile.
```

Comment:
At the time it seemed to me like a copy pasted description, the Utility and Makefile types should not be C/C++ project related right?